docs/src/main/asciidoc/security-getting-started.adoc

```diff
@@ -3,70 +3,70 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
-= Getting Started with Quarkus Security
+= Secure a Quarkus application with basic authentication and JPA
```

Comment:
Hello @sberyozkin,
Regarding the title discussion and comments from the last draft PR...

While the content helps you to get started, the more I read through the content, I felt that this topic was more of a tutorial quick start which explains how to secure a Quarkus application with basic authentication and JPA.
I thought that the content was a really great quick start tutorial for a learner more so than a Getting started guide.

I also wondered whether the 'Security Architecture and Guides' topic was a better fit for the Getting Started with Quarkus security guide. What do you think?

Reply:
Hi @michelle-purcell I'd like to retain a message here somehow that it is the first step for those new to security and to Quarkus security in particular to Get Started with Quarkus Security. This is why I'm not sure this title should be renamed since immediately in the following paragraph we say that here you can learn how to get stared with quarkus security quickly with basic authentication and JPA.
So I'd either keep `Getting Started wuth Quarkus Security` or may be tweak it a bit, example `Get Started with Quarkus Security using Basic Authentication` by not really highlighting `JPA` as Stuart had some concerns highlighting it too much until we get a reactive JPA support. JPA is really an implementation aspect of how to configure Quarkus with the user names/roles.

Can we agree on `Get Started with Quarkus Security using Basic Authentication` ?


>I also wondered whether the 'Security Architecture and Guides' topic was a better fit for the Getting Started with Quarkus security guide. What do you think?

IMHO 'Security Architecture and Guides'  should remain a document which briefly introduces everything related to Quarkus Security and links to the concrete documents expanding a given concept further

Reply:
@sberyozkin 

Thanks Sergey.

+1 to  **Get Started with Quarkus Security using Basic Authentication**. 

One question:

Both the Google and Red Hat style guidelines for headings prefer sentence case capitalization:

_In sentence-style, capitalize only the initial letter of the first word in the text and other words that require capitalization, such as proper nouns (product names)._

Are the following terms proper nouns in this context, i.e actual names of products/services/code/components products within Quarkus?
 
Quarkus Security 
Basic Authentication

If not, then I'd suggest that here and elsewhere we use lower case, for example:

**Get Started with security in Quarkus by using basic authentication**.

Reply:
Hi @michelle-purcell thanks, please change the title a bit more if you'd like, but yeah if we can retain a message somehow that it is about getting started then it would be not bad IMHO.

To me `Basic Authentication` is a recognizable concept, which is what the capitalization highlights, for example, `BA` is one acronym for it which is what Michal Maler found as well. So I'd prefer this particular approach here, as for example, once we move to OIDC, something like "oidc authentication" or "openid connect authentication" would not read well :-)
 
Perhaps, in such a title, it can be `Basic authentication` or in OIDC, `OpenId Connect`/`OIDC` `authentication`, may be it is a better approach, keep the authentication method capitalized such as `Basic` but the `authentication` itself - lower case...

Reply:
@sberyozkin Thank you Sergey for this context and information. It's really helpful.

I changed it to:

Get started with security in Quarkus by using Basic authentication

We are also discussing this amongst the team for consistency.

Reply:
@sberyozkin (cc @ebullient , @inoxx03 @gsmet ) 

After a bit more discussion on this...

For consistency, unless the 'Basic' bit of the phrase here is a proper noun (i.e a library name, command name, code name, product name, service name) grammatically it would make sense to use all lower case, as per the example below.

If we want to use the initialism (short form), we can do too. If an initialism/abbreviation is not commonly known, or if you are unsure whether it is commonly known, it is good practice to spell out the term at its first occurrence in the text body followed by the abbreviation in parentheses.

Therefore, to be consistent, could we consider using:

**Long:** basic authentication 
**Short:** BA
**First reference:**  basic authentication (BA)

Also, regarding your point about OIDC authentication, the first bit is a proper noun, so we can use uppercase as per the official standard/product. 

**Long:**  OpenID Connect authentication
**Shorter:** OIDC authentication
**Short:** OIDCA (ONLY use if this initialism does not duplicate an existing brand/trademark or commonly used technology standard in the industry/discipline of the content)
**First reference:** OpenID Connect authentication (OIDCA) 

What do you think?